Readme.MD

```diff
@@ -3,32 +3,67 @@
 ![https://travis-ci.com/mailchain/mailchain](https://img.shields.io/travis/com/mailchain/mailchain.svg?style=for-the-badge)
 ![https://coveralls.io/github/mailchain/mailchain](https://img.shields.io/coveralls/github/mailchain/mailchain.svg?style=for-the-badge)
 
-Mailchain is an email-like application. Using blockchain encryption and decentralized storage it delivers a simple, secure, email-like experience to blockchain users. Using Mailchain, Ethereum users can now send and receive rich-media HTML messages with attachments via an Ethereum address.
+## About Mailchain ###
+Extract from the [Mailchain specification](https://github.com/mailchain/mailchain-specification):
```

Comment:
This is refered to later and just takes up space at the top.

Reply:
Sorted.